client/src/pages/Apply/Apply.js

```diff
@@ -443,78 +443,60 @@ const Apply = ({ navigation }) => {
                   />
                 </View>
               </FormControl>
+
               <View style={styles.checkboxes}>
```

Comment:
On my phone, the Checkbox text doesn't wrap and it goes off the screen. I also don't see the code that fixed this two dev nights ago, which was setting `styles.checkboxes.width` to `95%` (unless you had another fix).